fastlane/lib/assets/completions/completion.fish

```diff
@@ -0,0 +1,37 @@
+# This function was taken from https://github.com/Carthage/Carthage/blob/master/Source/Scripts/carthage-fish-completion
+function __fish_prog_needs_subcommand
```

Comment:
IIRC fish doesn't deal gracefully with multiple functions being defined with the same name. I'd rename the function to something like `__fish_fastlane_needs_subcommand` to be sure.